packages/core/data_adapters/BaseAdapter.ts

```diff
@@ -81,8 +81,9 @@ export abstract class BaseAdapter {
     }
   }
 
+  /** @deprecated use `getConf(adapter, 'mySlotName')` instead */
   getConf(arg: string | string[]) {
```

Comment:
why deprecated? this is something I introduced and use it throughout most adapters. I don't believe the suggested syntax would work either, you would need `readConfObject(adapter.config, 'mySlotName')` since it's named config and not configuration

Reply:
if goal is to typecheck the names that are passed to the readConfObject functions, that would be cool, but not sure that it is possible to do this in most cases...many objects in the code are 'anyconfigurationmodels', 'anyconfigurationschemas'

Reply:
I suggest deprecating it because it can't do the same type checking as getConf and readConfObject.

Reply:
That work is going in #3636